### PR TITLE
CLC-7516, /roster view and header component; fix participating-depts in OEC widget

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-7516

Let's get it started.

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
       </div>
     </div>
     <div>
-      <Footer />
+      <Footer :show-build-summary="true" />
```
Comment (johncrossman) [inline src/views/Login.vue:23]: I assume we don't want to show build-info in LTI views.

File: src/views/StandaloneLti.vue [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <template>
   <div>
-    <ToolboxHeader />
```
Comment (johncrossman) [inline src/views/StandaloneLti.vue:3]: No toolbox-header in LTI views?

Comment (paulkerschen) [reply src/views/StandaloneLti.vue:3]: @johncrossman If we want to follow current Junction behavior, then this `StandaloneLti` view is equivalent to the current https://junction-dev.berkeley.edu/canvas/user_provision - what we use to look at LTI tools in a CalCentral (not LTI) context. Maybe this view needs a better name. Anyhow, in this context it’s probably useful to keep around the Junction-style tool header, footer build summary and so on.

When viewing LTI tools in the actual LTI context, we may need to use a completely different view (equivalent to the difference between `src/bcourses_embedded.html` and `src/index-junction.html` on the current master branch).

Comment (johncrossman) [reply src/views/StandaloneLti.vue:3]: Thanks @paulkerschen Makes sense. I'll revert that line.

File: src/views/Login.vue (hunk 1 above)
Comment (paulkerschen) [reply src/views/Login.vue:23]: What do you think about restoring the build summary here too? In the actual LTI context there won’t be a footer component at all.

Comment (johncrossman) [reply src/views/Login.vue:23]: @paulkerschen My comment was misplaced. We _are_ showing build-info in this case.

Comment (paulkerschen) [reply src/views/Login.vue:23]: Oh, right, of course.

**Pull request merged**